.travis/docker.mk

```diff
@@ -7,10 +7,10 @@ travis_start: ## Start containers stopped by `travis_stop`
 	docker-compose -f .travis/docker-compose-travis.yml start
 
 travis_test: ## Run tests on Docker containers, as on Travis CI
-	docker exec -it edx_notes_api env TERM=$(TERM) /edx/app/edx_notes_api/edx_notes_api/.travis/run_tests.sh
+	docker exec -it edx_notes_api env TERM=$(TERM) | TOX_ENV_DIR=$(TOX_ENV_DIR) /edx/app/edx_notes_api/edx_notes_api/.travis/run_tests.sh
```

Comment:
Hey @jmbowman Tox documentation says that it injects [`TOX_ENV_DIR` into the environment](https://tox.readthedocs.io/en/latest/config.html) and is working fine when I run tests locally with Tox, but on Travis it always has an empty value. Can you help me in figuring out this problem?

Reply:
You're not inside tox yet at this point; you're in a call to `make` directly from the Travis worker VM.  I think you want to use `TOXENV=$(TOXENV)` instead, since that's what was set as an environment variable by `.travis.yml`.

Reply:
I have added **`TOXENV`** variable instead now, and the environment being created is correct, but the build is failing when pytest command is executed with following trace.
```
tox -- python -Wd -m pytest
make: docker: Command not found
.travis/docker.mk:10: recipe for target 'travis_test' failed
make: *** [travis_test] Error 127
```
Can you help with this problem?

Reply:
Currently `make travis_test` calls tox, which then calls `make travis_test`.  I think instead you want tox to call `make validate`, which is what `make travis_test` originally did after all the docker and virtualenv setup.

Reply:
aah, Thanks.